bin/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Run script used by fluo-dev & fluo-deploy
```

Comment:
I looked at the source for fluo-deploy, I don't think it currently does.  But not 100%